articles/api/authentication/passwordless/start.md

```diff
@@ -0,0 +1,79 @@
+---
+  description: A description of the /passwordless/start Authentication API v2 endpoint listing its parameters.
+---
+
+# POST /passwordless/start
+
+Passwordless connections do not require the user to remember a password. Instead, another mechanism is used to prove identity, such as a one-time password sent every time the user logs in through email or SMS.
+
+## Email
+
+```JSON
+POST https://${account.namespace}/passwordless/start
+Content-Type: 'application/json'
+{
+  "client_id":   "{client-id}",
+  "connection":  "email",
+  "email":       "",
+  "send":        "",
+  "authParams":  
+}
+```
+
+Given a user's email address, this endpoint will send an email containing either:
+
+* A link (default, "send":"link"). You can then authenticate this user and they will be automatically logged in to your application. Optionally, you can append or override the link parameters (like scope, redirect_uri, protocol, response_type, etc.) using the `authParams` object.
+
+* A verification code ("send":"code"). You can then authenticate this user using the [/oauth/ro](/docs/api/authentication/oauth/ro) endpoint by specifying `email` as the `username` and `code` as the `password`.
+
+### Parameters
+
+| Parameter        | Type       | Description |
+|:-----------------|:-----------|:------------|
+| `client_id`      | string     | the `client_id` of your app |
+| `connection`     | string     | the name of an identity provider configured to your app |
+| `email`          | string     | the user's email address |
+| `send`           | string     | `link` (default) to send a URL or `code` to send a verification code |
+| `authParams`     | string     | |
```

Comment:
Is authParams a string or an object?
Should we list available authParams parameters here or is there a doc to link to with that info?